rosidl_adapter/rosidl_adapter/resource/__init__.py

```diff
@@ -14,10 +14,13 @@
 
 from io import StringIO
 import os
+from packaging import version
 import sys
 
 import em
 
+if version.parse(em.__version__) >= version.parse('4.0.0'):
+    from em import Configuration
```

Comment:
Rather than parsing the version, I'm wondering if we should instead attempt to import the thing we want.  Something like:

```
try:
    from em import Configuration
    em_has_configuration = True
except ImportError:
    em_has_configuration = False
```

And then below, if em_has_configuration, we use that (and the newer Interpreter API).  Otherwise, use the older API.  What do you think?